gravitee-apim-repository/gravitee-apim-repository-test/src/test/java/io/gravitee/repository/management/RoleRepositoryTest.java

```diff
@@ -169,4 +172,38 @@ public void shouldNotUpdateNull() throws Exception {
         roleRepository.update(null);
         fail("A null role should not be updated");
     }
+
+    @Test
+    public void shouldFindByIdAndOrganisationId() throws TechnicalException {
+        var id = "an_api_organisation_role";
+        Role role = roleRepository.findByIdAndReferenceIdAndReferenceType(id, REFERENCE_ID, REFERENCE_TYPE);
+        SoftAssertions.assertSoftly(soft -> {
+            soft.assertThat(role).isNotNull();
+            soft.assertThat(id).isEqualTo(role.getId());
+            soft.assertThat(REFERENCE_ID).isEqualTo(role.getReferenceId());
+            soft.assertThat(REFERENCE_TYPE).isEqualTo(role.getReferenceType());
```

Comment:
Usually, it is `assertThat(actual).isEqual(expected)`
I was wondering why we have the opposite here 🙈